sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2801,7 +2801,7 @@ SIREPO.app.directive('sbatchLoginModal', function() {
                     '<button  type="button" class="close" data-dismiss="modal"><span>&times;</span></button>',
                     '</div>',
                     '<div class="modal-body">',
-                        '<form>',
+                        '<form name="form">',
```

Comment:
"form" needs to be [unique](https://www.w3.org/TR/html4/struct/links.html#adef-name-A) so please name using context of the model eg. sbatchLoginModalForm.